v21.2/changefeed-sinks.md

```diff
@@ -140,10 +140,18 @@ Examples of supported cloud storage sink URIs:
 
 <a name ="cloud-parameters"></a>The following table lists the available parameters for cloud storage sink URIs:
 
-URI Parameter      | Description
--------------------+------------------------------------------------------------------
-`file_size`        | The file will be flushed (i.e., written to the sink) when it exceeds the specified file size. This can be used with the [`WITH resolved` option](create-changefeed.html#options), which flushes on a specified cadence. <br><br>**Default:** `16MB`
-`topic_prefix`     | Adds a prefix to all topic names.<br><br>For example, `CREATE CHANGEFEED FOR TABLE foo INTO 's3://...?topic_prefix=bar_'` would emit rows under the topic `bar_foo` instead of `foo`.
+URI Parameter      | Storage | Description
+-------------------+------------------------+---------------------------
+`AWS_ACCESS_KEY_ID` | AWS | The access key to your AWS account.
```

Comment:
nit: this is usually referred to as the "access key ID" in AWS and third party docs, tossing "ID" in there helps to  differentiate from the similarly-named "secret access key".

Reply:
Yeah, good point!